nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/RequestTracingService.java

```diff
@@ -274,13 +286,6 @@ public void bootstrapRequestTracingService() {
                         0, period, TimeUnit.SECONDS);
             }
             
-            if (executionOptions.getAdaptiveSamplingEnabled()) {
```

Comment:
This just moved up. I had a initialisation order issue causing a NPE that was caused by an exception earlier on and this fixed it but since that earlier exception should not be thrown in the first place it would not be strictly needed. Anyhow, I forgot to change it back and being earlier makes the code a bit more robust in case of failure so why not.